geomstats/special_orthogonal_group.py

```diff
@@ -342,23 +342,23 @@ def skew_matrix_from_vector(self, vec):
                 [[0., 1., 0.],
                  [-1., 0., 0.],
                  [0., 0., 0.]]
-                ])
+                ]]*vec.shape[0])
```

Comment:
Define n_vecs = vec.shape[0] since we're using it several times.

Reply:
I saw there was a gs function to get the shape (gs.shape(x)), however using that causes the tf backend to fail. I have resorted to n_vecs = vec.shape[0]. This works for both numpy and tensorflow